test/test_transforms_v2.py

```diff
@@ -186,7 +187,21 @@ def _check_functional_scripted_smoke(functional, input, *args, **kwargs):
         functional_scripted(input.as_subclass(torch.Tensor), *args, **kwargs)
 
 
-def check_functional(functional, input, *args, check_scripted_smoke=True, **kwargs):
+def _check_functional_torch_compile_smoke(functional, input, *args, **kwargs):
+    """Checks if the functional can be torch compiled and the compiled version can be called without error."""
+    if not isinstance(input, tv_tensors.Image):
```

Comment:
Should we just check for `type(input) == Tensor` directly?

Reply:
I just replicated the same check from `_check_functional_scripted_smoke`, however I agree that we can test all subclasses